mappings/net/minecraft/world/gen/decorator/AbstractConditionalPlacementModifier.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_6661 net/minecraft/world/gen/decorator/ConditionalDecorator
+CLASS net/minecraft/class_6661 net/minecraft/world/gen/decorator/AbstractConditionalPlacementModifier
```

Comment:
Why not just `ConditionalPlacementModifier`?

Reply:
The `Abstract` prefix would be consistent with `AbstractCountPlacementModifier`, and shows that it isn't a placement modifier itself, but is supposed to be used by other modifiers.